README-fr.adoc

```diff
@@ -0,0 +1,310 @@
+
+=== Autres options d'installation
+
+* {uri-install-doc}[Installation de l'outillage Asciidoctor]
+* {uri-install-osx-doc}[Installation d'Asciidoctor sur Mac OS X]
+
+== Utilisation
+
+Si la gem Asciidoctor s'est installée correctement, l'interface de ligne de commande (CLI) `asciidoctor` sera disponible dans votre PATH.
+Pour vérifier sa disponibilité, exécutez la commande suivante dans votre terminal :
+
+ $ asciidoctor --version
+
+Vous devriez voir les informations concernant la version d'Asciidoctor et celle de votre environnement Ruby s'afficher dans le terminal.
+
+[.output]
+....
+Asciidoctor 1.5.3 [http://asciidoctor.org]
+Runtime Environment (ruby 2.2.2p95 [x86_64-linux]) (lc:UTF-8 fs:UTF-8 in:- ex:UTF-8)
+....
+
+Asciidoctor fournit aussi une API.
+L'API permet une intégration avec d'autres logiciels Ruby, comme Rails, Sinatra et GitHub, ainsi que d'autres langages comme Java (via {uri-asciidoctorj}[AsciidoctorJ]) ou JavaScript (via {uri-asciidoctorjs}[Asciidoctor.js]).
+
+=== Interface de Ligne de Commande (CLI)
+
+La commande `asciidoctor` vous permet d'invoquer Asciidoctor à partir de la ligne de commande (càd, un terminal).
+
+La commande suivante convertit le fichier README.adoc en HTML et sauvegarde le résultat dans le fichier README.html dans le même dossier.
+Le nom du fichier HTML généré est tiré de celui du fichier source, l'extension a été changée pour `.html`.
+
+ $ asciidoctor README.adoc
+
+Vous pouvez contrôler le processeur Asciidoctor en ajouter plusieurs flags, vous pouvez en apprendre plus sur ces derniers en utilisant la commande :
+
+ $ asciidoctor --help
+
+Par exemple, pour écrire le fichier dans un répertoire différent, utilisez :
+
+ $ asciidoctor -D output README.adoc
+
+Le {uri-manpage}[man page] `asciidoctor` fournit une référence complète sur l'interface de ligne de commande.
+
+Référez-vous aux ressources suivantes pour en apprendre davantage sur la façon d'utiliser la commande `asciidoctor`.
+
+* {uri-render-doc}[Comment convertir un document ?]
+* {uri-themes-doc}[Comment utiliser la fabrique de feuilles de style Asciidoctor pour produire des thèmes personnalisés ?]
+
+=== API Ruby
+
+Pour utiliser Asciidoctor dans votre application, vous avez tout d'abord besoin de faire un require sur la gem :
+
+[source]
+require 'asciidoctor'
+
+Et vous pouvez convertir un fichier AsciiDoc en fichier HTML en utilisant :
+
+[source]
+Asciidoctor.convert_file 'README.adoc', to_file: true, safe: :safe
+
+WARNING: Quand vous utilisez Asciidoctor via l'API, le mode de sureté par défaut est `:secure`.
+Dans le mode secure, plusieurs fonctionnalités centrales sont désactivées, incluant la directive `include`.
+Si vous souhaitez activer ces fonctionnalités, vous aurez besoin de définir explicitement le mode de sureté avec une la valeur `:server` ou `:safe`.
+
+Vous pouvez aussi convertir une chaîne de texte en HTML embarquable (pour une insertion dans une page HTML) en utilisant :
```

Comment:
"embarquable" is weird :smile: 
Maybe "fragment" ? "partiel" ? "intégrable" ? or we could also write explicitly "en HTML contenant uniquement le corps du document"